common/usr/share/nethesis/NethServer/Module/Mail/Domain/Modify.php

```diff
@@ -75,6 +75,15 @@ public function validate(\Nethgui\Controller\ValidationReportInterface $report)
         ) {
             $report->addValidationErrorMessage($this, 'domain', 'valid_relay_notprimarydomain');
         }
+
+        // Domain validation, it must not be named like the first domain (FQDN)
+        $fqdn = strtolower($this->getPlatform()->getDatabase('configuration')->getType('DomainName'));
```

Comment:
DomainName is partial. FQDN is returned by PHP https://secure.php.net/manual/en/function.gethostname.php